src/explore-education-statistics-common/src/services/types/problemDetails.ts

```diff
@@ -0,0 +1,17 @@
+export interface ProblemDetails {
```

Comment:
`ProblemDetails` only seems to be used to be extended for `ValidationProblemDetails`, so does it need to exist separately and be exported?

Reply:
This mirrors how errors are modelled in the back end where `ProblemDetails` is the base error response type for .NET APIs.

Whilst we're not currently using this elsewhere, it's likely this will have some utility in some future case so it's a good idea to export it.